src/Domain/Service/Precondition/README.md

```diff
@@ -24,6 +24,4 @@ The reasoning behind the preconditions related to symlinks and hard links is not
 - **No soft links that point outside the codebase** ([`NoSymlinksPointOutsideTheCodebase`](NoSymlinksPointOutsideTheCodebaseInterface.php)) - Active and staging directories adjacent to one another on the filesystem have identical ancestors by definition. Hence, a relative link pointing out of one would target the same file as the corresponding link "next door" and, like previous scenarios, create the possibility of corrupting accidentally shared files. Links that point _within_ the codebase pose no threat--even if they cross internal package boundaries, i.e., Composer or other vendor libraries. The greatest risk in that case is probably links with missing targets, which seems improbable with low impact--it would represent an awfully bizarre bug on the part of the offending package and would pose no known danger to any Composer Stager operations.
 - **No soft links that point to a directory** ([`NoSymlinksPointToADirectory`](NoSymlinksPointToADirectoryInterface.php)) - Symlinks targeting directories are supported by [`RsyncFileSyncer`](../../../Infrastructure/Service/FileSyncer/RsyncFileSyncerInterface.php) but not yet by the [`PhpFileSyncer`](../../../Infrastructure/Service/FileSyncer/PhpFileSyncerInterface.php). Therefore they are forbidden when the latter is in use.
 
-Active and staging directories adjacent to one another on the filesystem have identical ancestors by definition. Hence, a relative link pointing out of one would target the same file as the corresponding link "next door" and, like previous scenarios, create the possibility of corrupting accidentally shared files. Links that point _within_ the codebase pose no threat--even if they cross internal package boundaries, i.e., Composer or other vendor libraries. The greatest risk in that case is probably links with missing targets, which seems improbable with low impact--it would represent an awfully bizarre bug on the part of the offending package and would pose no known danger to any Composer Stager operations.
```

Comment:
this sentence is repeated in the bullet list above

Reply:
Haha. Oops! 😛